sdk/core/azure-core/src/main/java/com/azure/core/http/policy/ExponentialBackoff.java

```diff
@@ -3,31 +3,55 @@
 
 package com.azure.core.http.policy;
 
+import com.azure.core.util.Configuration;
+import com.azure.core.util.CoreUtils;
 import com.azure.core.util.logging.ClientLogger;
+
 import java.time.Duration;
 import java.util.Objects;
 import java.util.concurrent.ThreadLocalRandom;
 
+import static com.azure.core.util.Configuration.PROPERTY_AZURE_REQUEST_RETRY_COUNT;
+
 /**
  * A truncated exponential backoff implementation of {@link RetryStrategy} that has a delay duration that exponentially
  * increases with each retry attempt until an upper bound is reached after which every retry attempt is delayed by the
  * provided max delay duration.
  */
 public class ExponentialBackoff implements RetryStrategy {
-
     private static final double JITTER_FACTOR = 0.05;
-    private static final int DEFAULT_MAX_RETRIES = 3;
+    private static final int DEFAULT_MAX_RETRIES;
     private static final Duration DEFAULT_BASE_DELAY = Duration.ofMillis(800);
     private static final Duration DEFAULT_MAX_DELAY = Duration.ofSeconds(8);
-    private final ClientLogger logger = new ClientLogger(ExponentialBackoff.class);
+
+    static {
+        String envDefaultMaxRetries = Configuration.getGlobalConfiguration().get(PROPERTY_AZURE_REQUEST_RETRY_COUNT);
+
+        int defaultMaxRetries = 3;
+        if (!CoreUtils.isNullOrEmpty(envDefaultMaxRetries)) {
+            try {
+                defaultMaxRetries = Integer.parseInt(envDefaultMaxRetries);
+                if (defaultMaxRetries < 0) {
+                    defaultMaxRetries = 3;
+                }
+            } catch (NumberFormatException ignored) {
+                new ClientLogger(ExponentialBackoff.class).verbose("{} was loaded but is an invalid number. "
```

Comment:
We can log this at `warning` level because the user's attempt to configure something did not work as expected.